package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "1inch-controller",
```

Comment:
@surajsingla333  Add the org name before the repo name.

@getsafle/1inch-controller